### PR TITLE
Using cennznet/api version 1.3.1 and polkadot/api 2.7.1

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Managed to fix all build issues, tested on local extrinsic, storage and toolbox(rpc).

### CODE REVIEW

File: packages/react-api/src/overrides/spec/edgeware.ts [hunk 1]
```diff
@@ -2,11 +2,10 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
```
Comment (holygits) [inline packages/react-api/src/overrides/spec/edgeware.ts:3]: Makes more sense to remove the edgeware stuff altogether

File: packages/react-query/src/BlockAuthors.tsx [hunk 2]
```diff
@@ -39,12 +39,12 @@ function BlockAuthors ({ children }: Props): React.ReactElement<Props> {
       let lastBlockNumber = '';
 
       // subscribe to all validators
-      api.query.session && api.query.session.validators((validatorIds): void => {
-        setValidators(validatorIds.map((validatorId): string => validatorId.toString()));
+      api.query.session && api.query.session.validators((validatorIds: any): void => {
```
Comment (holygits) [inline packages/react-query/src/BlockAuthors.tsx:42]: Maybe these types
`AccountId[]`
`Header`

File: packages/react-components/src/TreasuryProposal.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export default function TreasuryProposal ({ className, asInset, insetProps, onCl
     if (!proposal && proposalId) {
       api.query.treasury
         .proposals<Option<TreasuryProposalType>>(proposalId)
-        .then((proposal): TreasuryProposalType | null => proposal.unwrapOr(null))
+        .then((proposal: any): TreasuryProposalType | null => proposal.unwrapOr(null))
```
Comment (holygits) [inline packages/react-components/src/TreasuryProposal.tsx:35]: Seems odd that `TreasuryProposalType` doesn't work...

File: packages/react-components/src/InputRpc/CennznetJsonRpc.ts [hunk 4]
```diff
@@ -1,27 +1,25 @@
 import jsonrpc from '@polkadot/types/interfaces/jsonrpc';
-import {DefinitionRpcSub, DefinitionRpcParam, DefinitionTypeType} from '@cennznet/types';
+import {DefinitionRpcSub } from '@cennznet/types';
 import {DefinitionRpcExt} from '@polkadot/types/types';
-import cennznetBare from '@cennznet/api/rpc';
+import * as definitions from '@cennznet/types/interfaces/definitions';
```
Comment (holygits) [inline packages/react-components/src/InputRpc/CennznetJsonRpc.ts:4]: can you explain this code alittle more?
why 'cennzx' ? + I think it's still 'cennzxSpot' in v36

**Review by holygits — COMMENTED** [review]
Nice, few small things but otherwise lgtm!

File: packages/react-query/src/BlockAuthors.tsx (hunk 2 above)
Comment (KarishmaBothara) [reply packages/react-query/src/BlockAuthors.tsx:42]: I tried with those, I got error.. even polkadot/apps do not provide any types for these..

File: packages/react-api/src/overrides/spec/edgeware.ts (hunk 1 above)
Comment (KarishmaBothara) [reply packages/react-api/src/overrides/spec/edgeware.ts:3]: sure.. will remove Edgware completely.. anyways its doesn't help us

File: packages/react-components/src/InputRpc/CennznetJsonRpc.ts (hunk 4 above)
Comment (KarishmaBothara) [reply packages/react-components/src/InputRpc/CennznetJsonRpc.ts:4]: rpc is cennzx even in rv36...
so.. If you check this code
https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/jsonrpc.ts
here they have used an example type to be used for example babe (consider type as babe).. we don't have babe, we have cennzx, genericAsset for which RPC calls are available.. I have tested ga RPC calls too.. They are working fine..

Comment (holygits) [reply packages/react-components/src/InputRpc/CennznetJsonRpc.ts:4]: I see, normally good to leave a comment on something unusual like this to save the next person some time understanding why it's like that

**Review by holygits — APPROVED** [review]

**Pull request merged**